x-pack/plugins/lists/server/services/exception_lists/utils/import/create_exceptions_stream_from_array.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { pipe as lodashPipe } from 'lodash/fp';
```

Comment:
I wouldn't rename pipe with an `as lodashPipe` unless it's clashing with `pipe` somewhere here.

Also, do you want to use lodash pipe or `import { pipe } from 'fp-ts/lib/pipeable';`? I'm seeing more of the `fp-ts` being used these days.

Reply:
Updated to use the pipe I was already pulling in from `fp-ts/lib/pipeable`. Thanks!